cpp/tests/utilities/column_utilities.cu

```diff
@@ -136,6 +137,19 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+// copy column bitmask to host (used by to_host())
+std::vector<bitmask_type> bitmask_to_host(cudf::column_view const& c) {
+  std::vector<bitmask_type> host_bitmask;
```

Comment:
Originally, I coded this using an IILE (Immediately Invoked Lambda Expression) to write a more declarative piece of code and avoid having to use `.resize`, but then realized you don't even need the lambda:
```cpp
    if (c.nullable()) {
      auto num_bitmasks = bitmask_allocation_size_bytes(c.size()) / sizeof(bitmask_type);
      std::vector<bitmask_type> = host_bitmask(num_bitmasks);
      CUDA_TRY(cudaMemcpy(host_bitmask.data(), c.null_mask(), num_bitmasks * sizeof(bitmask_type),
                          cudaMemcpyDeviceToHost));
      return host_bitmask;
    } else {
      return std::vector<bitmask_type>{};
    }
```
I try avoid writing code that declares `x`, modifies it based on conditionals and then returns it after conditionals. The more declarative the better.